core/storage/suggestion/gae_models_test.py

```diff
@@ -93,6 +93,66 @@ def setUp(self) -> None:
             'reviewer_2', self.change_cmd, self.score_category,
             'exploration.exp1.thread_5', None)
 
+    def test_get_model_association_to_user(self) -> None:
+        gen_model = suggestion_models.GeneralSuggestionModel
+        self.assertEqual(
+            gen_model.get_model_association_to_user(),
+            base_models.MODEL_ASSOCIATION_TO_USER.MULTIPLE_INSTANCES_PER_USER
+        )
+
+    def test_get_export_policy(self) -> None:
+        dictionary = {
```

Comment:
Please use a more descriptive name for this, e.g. expected_export_policy_dict.

Also, alphabetize the keys in the dict -- it makes it easier in the future to see if something is missing, or to add a new key.

Ditto elsewhere.

Reply:
Resolved this.